specs/cb_info.table

```diff
@@ -0,0 +1,26 @@
+table_name("cb_info")
```

Comment:
Can we name this `carbon_black_info`? Feel free to using something like:
```python
table_name("carbon_black_info", aliases=["cb_info"])
```

Then you can `select * from cb_info` but it wont show up in the `.tables` (essentially it creates a `VIEW`).